packages/common-ui-web/src/components/VehicleTypeSelection/utils.ts

```diff
@@ -40,3 +42,21 @@ export function getInitialSelectedVehicleType(
   }
   return vehicleTypes[Math.floor(vehicleTypes.length / 2)];
 }
+
+export function getVehicleTypeFromInspection(
+  inspection: GetInspectionResponse,
+): VehicleType | undefined {
+  const imageWithSightId = inspection.entities.images.find((image) => image.sightId);
+  if (!imageWithSightId) {
+    return undefined;
+  }
+
+  const sight = Object.values(sights).find((s) => s.id === imageWithSightId.sightId);
+  if (!sight) {
+    return undefined;
+  }
+
+  const vehicle = Object.values(vehicles).find((v) => v.id === sight.vehicle);
+
+  return vehicle?.type;
```

Comment:
Double check that all vehicle types in `vehicle.json` match the `VehicleType` enum